nuxt.config.ts

```diff
@@ -8,7 +7,7 @@ export default {
    ** Headers of the page
    */
   head: {
-    title: pkg.name, // TODO change page title
+    title: '%s - PROJECT_NAME', // TODO change PROJECT_NAME
```

Comment:
If a page title isn't set, then this will make the page title be "[blank space] - PROJECT_NAME".
Instead of `title`, use `titleTemplate` which lets you explicitly check if the current page's title property is set:
```
titleTemplate: pageTitle => (pageTitle ? `${pageTitle} - ` : '') + 'PROJECT NAME', // TODO change PROJECT_NAME
```